tests/services/logging.service.spec.ts

```diff
@@ -22,7 +22,7 @@ const mockDates = () => {
 const initializeLoggingService = () => {
   const electronService = jasmine.createSpyObj('ElectronService', ['isElectron']);
```

Comment:
Can we remove this?

Reply:
yes, seems like i forgot to remove it, i'll remove it now